src/Idephix/Extension/Slack/Slack.php

```diff
@@ -0,0 +1,88 @@
+<?php
+
+namespace Idephix\Extension\Slack;
+
+use Idephix\IdephixInterface;
+use Idephix\Extension\IdephixAwareInterface;
+
+/**
+ * Description of Slack wrapper
+ *
+ * @author dymissy
+ */
+class Slack implements IdephixAwareInterface
+{
+    /**
+     * @var \Idephix\IdephixInterface
+     */
+    private $idx;
+
+    private $settings;
+
+    public function __construct($args = array())
+    {
+        $defaults = array(
+            'url' => '',
+            'channel' => '#general',
+            'icon_url' => 'https://slack.com/img/icons/app-57.png',
+            'username' => 'slackbot'
+        );
+
+        $this->settings = array_merge($defaults, $args);
+    }
+
+    public function sendToSlack($message, $attachments = array(), $channel = '', $icon_url = '', $username = '')
+    {
```

Comment:
Are we sure we want to expose all these parameters ? The final user can set all these data in the ```$config``` array
What the ```$attachment``` parameter is supposed to contain? In the ```idxfile.php.dist``` we have 

```php
array(
 array(
  'fallback' => "Hi guys, I'm Idephix and this is my first slack message!",
  'pretest'  => "Hi guys, I'm Idephix and this is my first slack message!",
  'color'    => "#43e17b",
  'fields'   => array(
  array(
    'title' => 'I\'m Idephix',
    'value' => "Hello world, aren't you using Idephix?\nWhy not? Please <http://getidephix.com|download Idephix> right now",
   'short' => false
      )
    )
) 
```

Moreover but I see a couple more issues in the $attachment param:
1. $attachment  is perhaps the wrong name
2. the data passed is extremely complicated
3. how the final user is supposed to know the right structure to pass?